src/app/ui/learn/LearnPageArticlesSection/LearnPageArticlesSectionClient.tsx

```
@@ -0,0 +1,35 @@
+'use client';
+
+import { SectionCard } from '@/components/Cards/SectionCard/SectionCard';
+import type { FC } from 'react';
+import { useLearnFiltering } from '../../../../providers/LearnProvider/filtering/LearnFilteringContext';
+import Stack from '@mui/material/Stack';
+import { LearnPageArticlesPagination } from './LearnPageArticlesPagination';
+import { LearnPageArticlesFilteringBar } from './LearnPageArticlesFilteringBar';
+import { LearnPageArticlesList } from './LearnPageArticlesList';
+
+interface LearnPageArticlesSectionClientProps {}
```

Biome (2.4.7)

[error] An empty interface is equivalent to {}.

(lint/suspicious/noEmptyInterface)